assets/components/prometheus-operator/manifests/templates/grafana/dashboards-1.14/persistentvolumesusage.yaml

```diff
@@ -566,4 +566,4 @@ data:
         "uid": "919b92a8e8041bd567af9edab12c840c",
         "version": 0
     }
-{{- end }}
\ No newline at end of file
+{{- end }}
```

Comment:
This we carry from upstream, why change it?

Reply:
Because the data is not correct.

Reply:
Yeah, but that should be fixed upstream and does not need to be changed on our side. To minimize the diff with upstream.

Reply:
I think we are way out of sync with upstream and cannot move to latest upstream because of the PVC issue, I don't see any immediate solution to the problem. I myself don't like such changes since it is harder to find out which are our changes and which are upstream inclusions in the plethora of diff.

Reply:
I'm not sure I understand. @surajssd are you saying that without this line change things are broken? In that case I think we can fix it here and then try to contribute upstream.

If the change does not break things we should just leave it.